apps/leo_manager/priv/leo_manager_0.conf

```diff
@@ -86,9 +86,25 @@ consistency.rack_aware_replicas = 0
 ## A number of replication targets
 ## mdc_replication.max_targets = 2
 
-## A number of replicas a DC
+## A number of replicas per a datacenter
+## [note] A local LeoFS sends a stacked object which contains an items of a replication method:
+##          - [L1_N] A number of replicas
+##          - [L1_W] A number of replicas needed for a successful WRITE operation
+##          - [L1_R] A number of replicas needed for a successful READ operation
+##          - [L1_D] A number of replicas needed for a successful DELETE operation
+##       A remote cluster of a LeoFS system which receives its object,
+##       and then replicates it by its contained reoplication method.
```

Comment:
The constrain **N_Value >= MDCR_N_Value** discussed on
- https://github.com/leo-project/leo_redundant_manager/pull/4#discussion_r111337958
- https://github.com/leo-project/leo_redundant_manager/pull/4#discussion_r111338679

also should be noted here? like

mdc_replication.max_targets on the source cluster must be less than/equal 
consistency.num_of_replicas on the destination cluster.

otherwise MDC replication will fail at https://github.com/leo-project/leo_redundant_manager/blob/1.9.36/src/leo_redundant_manager_api.erl#L638.